src/Sylius/Component/Core/spec/OrderProcessing/OrderShipmentProcessorSpec.php

```diff
@@ -62,6 +64,15 @@ function it_creates_a_single_shipment_with_default_shipping_method_and_assigns_a
 
         $order->isShippingRequired()->willReturn(true);
 
+        $productVariant1->isShippingRequired()->willReturn(true);
+        $productVariant2->isShippingRequired()->willReturn(true);
+
+        $itemUnit1->getShippable()->willReturn($productVariant1);
+        $itemUnit1->getShipment()->willReturn(null);
+
+        $itemUnit2->getShipment()->willReturn(null);
+        $itemUnit2->getShippable()->willReturn($productVariant2);
```

Comment:
Same code snippets written differently -- let's make them identical.

```suggestion
        $itemUnit1->getShippable()->willReturn($productVariant1);
        $itemUnit1->getShipment()->willReturn(null);

        $itemUnit2->getShippable()->willReturn($productVariant2);
        $itemUnit2->getShipment()->willReturn(null);
```